src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/ReindexJobTask.cs

```diff
@@ -393,6 +393,7 @@ private async Task<ReindexJobQueryStatus> ProcessQueryAsync(ReindexJobQueryStatu
                     await jobSemaphore.WaitAsync();
                     try
                     {
+                        _logger.LogInformation($"Reindex job updating progress, current result count: {0}", results.Results.Count());
```

Comment:
>0 [](start = 103, length = 1)

nit: you can put  {count} and that should be properly displayed in logs, AFAIK. Also no need for string interpolation.